.github/workflows/ci.yml

```diff
@@ -52,7 +52,11 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
-      - name: install dependencies
+      - if: matrix.node-version == 12
+        name: install dependencies
+        run: yarn install --frozen-lockfile --ignore-engines
```

Comment:
Why are we adding this exception for to ignore the engine check for Node 12? That defeats the check to ensure we remain compatible with Node 12.

It looks like @scalvert/bin-tester has a dependency that is not compatible with Node 12. That is where I think we should fix the issue.

Reply:
Because firstly it's a devDependency, not a dependency, so it's not material whether Node 12 is supported in that dependency or not. The incompatibility is in `fixturify-project`, and that's not going to be downgraded since it was explicitly updated to make new APIs available to `bin-tester`, which in turn are required by `ember-template-lint`. 

Adding this escape hatch to CI simply avoids this issue for our tests, but doesn't negatively impact any code destined for the published package. This coupled with the short duration that we have to support Node 12 felt like an acceptable tradeoff.

Reply:
Ya I appreciate what you're saying, @bmish. I also appreciate all the work you've done on the repo in the last while!!

I do agree there's some potential risk associated with this. I feel like it's inline with the risk associated with skipping those two test files for Node 16 - it's not ideal, but intended to be short-lived. 

I'll see if I can get CI back up and running, and for now I've separated Node 12 into its own job so that it can be easily removed once we remove it.

Reply:
Sounds good, thanks!